test/Hamiltonians.jl

```diff
@@ -1213,6 +1213,28 @@ end
         @test issymmetric(even_m) # because it was passed through `fix_approx_hermitian!`
         @test even_sm ≈ even_m # still approximately the same!
     end
+
+    @testset "basis-only" begin
+        m = 5
+        n = 5
+        add = near_uniform(BoseFS{n,m})
+        ham = HubbardReal1D(add)
+        @test_throws ArgumentError build_basis_only_from_LO(ham, BoseFS((1,2,3))) # wrong address type
+        # same basis as BSR
+        bsr = BasisSetRep(ham)
+        basis = build_basis_only_from_LO(ham)
+        @test basis == bsr.basis
+        # sorting
+        bsr = BasisSetRep(ham, add; sort = true)
+        basis = build_basis_only_from_LO(ham, add; sort = true)
+        @test basis == bsr.basis
```

Comment:
```suggestion
        basis = build_basis_only_from_LO(ham, add; sort = true)
        @test basis == sort!(bsr.basis)
```
Just to shave a few milliseconds off the time it takes to run the tests ...